compose/config/config.py

```diff
@@ -199,14 +238,52 @@ def load(config_details):
 
     Return a fully interpolated, extended and validated configuration.
     """
+    version = get_config_version(config_details)
+    if version not in COMPOSEFILE_VERSIONS:
+        raise ConfigurationError('Invalid config version provided: {0}'.format(version))
```

Comment:
By checking `COMPOSEFILE_VERSIONS` here we now have the source of truth for "what versions do we support" in two places, below in the if/elif, and in the const.

What do you think about removing this check and moving the error to an `else` in the branch below ?